src/extra.ml

```diff
@@ -61,11 +64,16 @@ module Option =
       | None    -> ()
       | Some(e) -> f e
 
-    let get : 'a option -> 'a -> 'a = fun o d ->
+    let get_default : 'a -> 'a option -> 'a = fun d o ->
       match o with
       | None    -> d
       | Some(e) -> e
 
+    let get : 'a option -> 'a = fun o ->
+      match o with
+      | None    -> assert false
+      | Some(e) -> e
+
```

Comment:
I don't think this change is a good idea for two reasons:
 - `get` is a standard name for your `get_default`,
 - we probably don't want to have your `get` function in the interface. Such functions should be avoided since they can lead to tricky problems. It is by far better to inline it when used, which forces you to say why the `None` case is not accessible.

Reply:
Fixed in bc53195065b3db652ac59f271d627e03e9c7ab59